src/main/dialogs/waitDialog.js

```diff
@@ -118,8 +119,17 @@ gpii.app.waitDialog.hide = function (that) {
     if (remainingDisplayTime > 0) {
         that.dismissWaitTimeout = setTimeout(function () {
             that.dialog.hide();
+            gpii.app.waitDialog.toggleAnimation(that, false);
         }, remainingDisplayTime);
     } else {
         that.dialog.hide();
+        gpii.app.waitDialog.toggleAnimation(that, false);
     }
 };
+
+gpii.app.waitDialog.toggleAnimation = function (that, animate) {
+    var script = fluid.stringTemplate("jQuery(document.body).toggleClass(\"animate\", %animate)", {
+        animate: animate
+    });
+    that.dialog.webContents.executeJavaScript(script);
```

Comment:
We've tried to keep the main and the renderer processes as decoupled as possible using IPC and leaving changes to the DOM  to be done by the renderer process but here it's probably a reasonable exception as the `waitDialog` doesn't have a `.js` file on its own.

Reply:
Yes, I completely agree with you. I tried to keep it all in the renderer, however I couldn't find a way to detect if the dialog had been opened.